src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj

```diff
@@ -5,7 +5,7 @@
     <Version>1.0.3</Version>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <PackagePlatforms>x64;x86;arm</PackagePlatforms>
+    <PackagePlatforms>x64;x86;arm64;arm</PackagePlatforms>
```

Comment:
I dont think it claims to support arm per https://github.com/dotnet/buildtools/blob/3c700d1ecc358c6c1b3ffbaaeb49e7864a8e9ba7/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L125 which expects a trailing ";" to enable the platform.

That said, I will be looking into enabling Arm32 once I am done with this.